lcmtypes/lcmt_atlas_command.lcm

```diff
@@ -0,0 +1,58 @@
+package drake;
```

Comment:
I was originally thinking that, yeah. But in our discussions today we decided to move all the DRC-lcm related stuff into DRC, so this lcm type will be gone again in a couple of days (since it just duplicates the DRC type)